packages/models-library/src/models_library/services_creation.py

```diff
@@ -36,10 +41,18 @@ class CreateServiceMetricsAdditionalParams(BaseModel):
                 "user_email": "test@test.com",
                 "project_name": "_!New Study",
                 "node_name": "the service of a lifetime _ *!",
-                "service_key": ServiceKey("simcore/services/dynamic/test"),
-                "service_version": ServiceVersion("0.0.1"),
+                "service_key": ServiceKeyTypeAdapter.validate_python(
+                    "simcore/services/dynamic/test"
+                ),
+                "service_version": ServiceVersionTypeAdapter.validate_python("0.0.1"),
                 "service_resources": {},
                 "service_additional_metadata": {},
+                "pricing_unit_cost_id": None,
             }
         }
     )
+
+
+CreateServiceMetricsAdditionalParamsTypeAdapter = TypeAdapter(
```

Comment:
THOUGHT: i find more readable the composed name instead of ending up having many huge alias names. 
Similar situation as with `Page[Model]` rather than an alias `PageModel`

Reply:
I have the same opinion as you but people pointed out the "performance" hit in instantiating the objects. So we came up with this alternative. We have to solutions:
1. stick with `TypeAdapter(Model)`
2. go with this proposal for optimisation reason